### PR TITLE
Add external model's example code and Docs.

### PR DESCRIPTION
**Pull request by ziqi-jin**
- nanodet_plus code
- yolor code
- yolov6 code
- yolov5lite code
- yolox code
- scaled yolov4 code
- retinaface code
- scrfd code
- ultraface code
- yolov5face code
- arcface code
- cosface code
- partial_fc code
- vpl code
- modnet code
vis_result-nano

vis_result-yolov6
vis_result-yolox

vis_result-retina

vis_result-scrfd
vis_result-ultra

vis_result-yolov5face

vis_result-modnet

test_lite_matting_input

vis_result-scaledyolov4
vis_result-yolor

vis_result-v5lite

visualized_result_yolov5

vis_result-yolov7

visualized_result-m
visualized_result_fg-m

test_lite_matting_input-m

test_lite_matting_bgr-m

visualized_result_replaced_bg1024

visualized_result_fg1024

visualized_result

visualized_result_fg

### CODE REVIEW

File: examples/vision/detection/nanodet_plus/python/infer.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+args = parse_arguments()
+
+# 配置runtime，加载模型
+runtime_option = build_option(args)
+model = fd.vision.detection.NanoDetPlus(
+    args.model, runtime_option=runtime_option)
+
+# 预测图片检测结果
+im = cv2.imread(args.image)
+result = model.predict(im)
```
Comment (DefTruth) [inline examples/vision/detection/nanodet_plus/python/infer.py]: 改成predict(im.copy())，其他案例同理

Comment (DefTruth) [inline examples/vision/detection/nanodet_plus/python/infer.py]: 在predict之后加一句print(result)，其他案例同理

**Review by DefTruth — CHANGES_REQUESTED** [review]

**Comment by jiangjiajun** [comment]
f0d213d3bbf4b008f5ee3eb4127be787

**Comment by DefTruth** [comment]
## YOLOv7End2EndORT  
image

**Comment by DefTruth** [comment]
## YOLOv7End2EndTRT
image

**Pull request merged**